examples/web-wagmi/src/profiles/UseUpdateProfileImage.tsx

```diff
@@ -63,7 +63,7 @@ function UploadAndUpdateNewImage({ profile }: { profile: ProfileFieldsFragment }
     error: updateError,
     isPending,
   } = useUpdateProfileImage({
-    profileId: profile.id,
+    profile,
   });
```

Comment:
nit: do we need the wrapping object at all or can we pass `profile` directly?

Reply:
personally prefer the wrapping object for the API that's used externally as it's easier to add new fields later without introducing a breaking change. It appears it's also consistent with the hook that we already have: `useUpdateDispatcherConfig`, `useUnfollow`, and others

Reply:
I think some... the ones about publication do accept the object directly.